specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-10-01/mfe.json

```diff
@@ -15485,7 +15485,7 @@
       "type": "object",
       "properties": {
         "authMode": {
-          "description": "[Required] Use 'Key' for key based authentication and 'AMLToken' for Azure Machine Learning token-based authentication. 'Key' doesn't expire but 'AMLToken' does.",
+          "description": "[Required] The authentication method for invoking the endpoint (data plane operation). Use 'key' for key-based authentication. Use 'AMLToken' for Azure Machine Learning token-based authentication. Use 'AADToken' for Microsoft Entra token-based authentication.",
```

Comment:
Did you mean to change the capitalization of 'Key' to 'key'? It is still declared as 'Key' in the enum definition.

Reply:
fixed this